### PR TITLE
Making the README.md file

### PR DESCRIPTION
**Pull request by obaidofj**
this PR for making the readme file which describe the reapo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
-# etch-a-sketch_Obaid
\ No newline at end of file
+# etch a sketch Obaid Project
+
+#WHY 
+These files here are project for practising what we learned in Gaza Sky Geeks Web Programing Fundemental Course
+
+#WHAT 
+this project is to make etch a sketch so as to be able to draw on the page by moving mosue over, painting by random color/colors or py selecting a color
```
Comment (Jood80) [inline README.md]: *mouse & *by :eyes:

File: README.md [hunk 2]
```diff
@@ -1 +1,10 @@
...
+
+#HOW 
+this project created by HTML CSS and Javascript, as i created divs dynamicly using javascript and when the mose hover the color of the divs change and we can select by which colors we can paint.
```
Comment (Jood80) [inline README.md]: this is good :ok_hand:  but here what we meant with "How" is from a user perspective which is the user journey.
for instance:
>  as a user, I would be able to erase the colored grids.

whenever you add a new feature, you may add its own user journey.
PS: make it only one sentence each, specific and simple.

File: README.md (hunk 1 above)
Comment (obaidofj) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (obaidofj) [reply README.md]: ok

File: README.md [hunk 3]
```diff
@@ -4,7 +4,10 @@
...
 #WHAT 
-this project is to make etch a sketch so as to be able to draw on the page by moving mosue over, painting by random color/colors or py selecting a color
+this project is to make etch a sketch so as to be able to draw on the page by moving mouse over, painting by random color/colors or by selecting a color
 
 #HOW 
-this project created by HTML CSS and Javascript, as i created divs dynamicly using javascript and when the mose hover the color of the divs change and we can select by which colors we can paint. 
+the user can paint by moving the mose.
+the user can select by which color to paint.
+or he can paint by random colors.
+the user will be able to erase the colors.
```
Comment (Jood80) [inline README.md:13]: This one is better :+1:

**Review by Rahaf-96 — APPROVED** [review]

**Review by saharAdem — APPROVED** [review]

**Pull request merged**